homeassistant/components/leviosa_shades/cover.py

```diff
@@ -0,0 +1,141 @@
+"""The Leviosa Shades Zone base entity."""
+import logging
+
+from leviosapy import LeviosaShadeGroup as tShadeGroup, LeviosaZoneHub as tZoneHub
+import voluptuous as vol
+
+from homeassistant.components import cover
+from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv, entity_platform
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    BLIND_GROUPS,
+    DOMAIN,
+    MANUFACTURER,
+    MODEL,
+    SERVICE_NEXT_DOWN_POS,
+    SERVICE_NEXT_UP_POS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+# Estimated time it takes to complete a transition
+# from one state to another
+TRANSITION_COMPLETE_DURATION = 30
+PARALLEL_UPDATES = 1
+
+COVER_NEXT_POS_SCHEMA = vol.Schema({vol.Optional(ATTR_ENTITY_ID): cv.entity_ids})
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: cover.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Leviosa shade groups."""
+    _LOGGER.debug(
+        "Setting up %s[%s]: %s",
+        entry.domain,
+        entry.title,
+        entry.entry_id,
+    )
```

Comment:
Do you really need this debug logging? By default there is already debug logging around setting up new platforms. I would try to keep debug logging to what is required to debug this integration.

Reply:
I will remove them

Reply:
Suggestion accepted, thanks.